main.py

```diff
@@ -147,4 +168,4 @@ def handler(request, event):
 
 
 if __name__ == "__main__":
-    handler(None, None)
+    handler(None, None)
```

Comment:
```suggestion
    handler(None, None)

```

Reply:
Don't get the suggestion? An extra blank line?